src/system.js

```diff
@@ -6,7 +6,7 @@
  */
 
 // Define contexts to require
-const contexts = [require.context("@/components/", true, /\.vue$/)]
+const contexts = [require.context("./components/", true, /\.vue$/)]
```

Comment:
@LukasHirt couldn't make this one work properly (guess it's a Webpack thing), maybe you can shed some lights on it?

Reply:
`@` is an alias created in webpack which is equal to the `src` path.

https://github.com/owncloud/owncloud-design-system/blob/917a6c8c54191c80c35d6aabb9314b03b61218f8/build/webpack.base.conf.js#L30

Reply:
Since this doesn't work as expected and we're about to switch from Webpack to Rollup we've decided not to spend further effort on resolving it for now